src/logic/safe/store/selectors/index.ts

```diff
@@ -208,6 +208,14 @@ export const safeModulesSelector = createSelector(safeSelector, safeFieldSelecto
 
 export const safeFeaturesEnabledSelector = createSelector(safeSelector, safeFieldSelector('featuresEnabled'))
 
+export const safeOwnersAddressesSelector = createSelector(safeOwnersSelector, (owners): string[] => {
+  if (!owners) {
+    return []
+  }
+
+  return owners?.map((o) => o.address)?.toArray()
```

Comment:
the `o` thing... WDYT about `.map(({ address }) => address)` instead?

Or it won't hurt anybody if you just call it `owner`, it will be clearer

Reply:
it's done